### ISSUE TITLE
Global namespace

### ISSUE THREAD

**Issue by mariusmuja** [OutWatch/outwatch#124]
This has been discussed before in #32, but I'd like to bring up the issue again.

Currently, with the inclusion of `scala-dom-types` there are even more identifiers (html tags, attributes, events + css style names) imported into the global namespace by `import outwatch.dom._`. I think this is likely to cause issues along the way.

How do people feel about breaking compatibility and moving all dom identifiers to `outwatch.dom.all._`, so the name explicitly shows that *everything* gets imported and have `import outwatch.dom._` only import `Attributes`, `Tags`, `Styles`, `Handler`...

**Comment by raquo** [comment]
So... what would be in `dom.all` but not in `dom`?

**Comment by mariusmuja** [comment]
> So... what would be in dom.all but not in dom?

 `import outwatch.dom.all_` would bring in all html tags, attributes, styles, events, essentially what `import outwatch.dom._` does now and `import outwatch.dom._` would only import the `Attributes`, `Tags`, `Styles`... objects, so one could do
```scala
import outwatch.dom.{Attributes => A, Tags => T, Styles => S}

T.div( A.id := "some-id")
```

I realise that it's currently possible to do `import outwatch.dom.{Attributes => A, Tags => T, Styles => S}` and not use `import outwatch.dom._`, but I'm guessing that it's not what typically users are doing and having the `all` in the import makes it more obvious that *all* identifiers get imported, so it should be "handled with care" (maybe used only locally where the VDom is constructed).

**Comment by cornerman** [comment]
> I realise that it's currently possible to do import outwatch.dom.{Attributes => A, Tags => T, Styles => S} and not use import outwatch.dom._,

Yes, but then you will need the implicits for working with `VDomModifier` and `VNode`. Therefore, I agree with this proposal. We should have an import `outwatch.dom._`, which we can advise people to use without filling up the scope so aggressively. They can still explicitly import all or build up a more fine granular scheme.

**Comment by mariusmuja** [comment]
> Yes, but then you will need the implicits for working with VDomModifier and VNode.

Yes, good point!

**Comment by mariusmuja** [comment]
If we decide to break compatibility like this, this would be an easy migration command for most situations:
```bash
find src -name '*.scala' | xargs sed -i -e 's/^\([ \t]*\)import outwatch.dom._/\1import outwatch.dom._\n\1import outwatch.dom.all._/'
```

**Comment by mariusmuja** [comment]
Fixed by #132.
